content/learn/book/migration-guides/0.7-0.8/_index.md

```diff
@@ -0,0 +1,514 @@
++++
+title = "0.7 to 0.8"
+weight = 4
+sort_by = "weight"
+template = "book-section.html"
+page_template = "book-section.html"
+insert_anchor_links = "right"
+[extra]
+long_title = "Migration Guide: 0.7 to 0.8"
++++
+
+<!-- Github filter used to find the relevant PRs "is:pr label:C-Breaking-Change closed:>2022-04-15 [Merged by Bors]" -->
+
+### [Camera Driven Rendering](https://github.com/bevyengine/bevy/pull/4745)
+
+<!-- pls cart add a migration guide section :( -->
+
+This is a very complicated change and it is recommended to read the linked PRs for more details
+
+```rust
+// old 3d perspective camera
+commands.spawn_bundle(PerspectiveCameraBundle::default())
+
+// new 3d perspective camera
+commands.spawn_bundle(Camera3dBundle::default())
+```
+
+```rust
+// old 2d orthographic camera
+commands.spawn_bundle(OrthographicCameraBundle::new_2d())
+
+// new 2d orthographic camera
+commands.spawn_bundle(Camera2dBundle::default())
+```
+
+```rust
+// old 3d orthographic camera
+commands.spawn_bundle(OrthographicCameraBundle::new_3d())
+
+// new 3d orthographic camera
+commands.spawn_bundle(Camera3dBundle {
+    projection: OrthographicProjection {
+        scale: 3.0,
+        scaling_mode: ScalingMode::FixedVertical,
+        ..default()
+    }.into(),
+    ..default()
+})
+```
+
+UI no longer requires a dedicated camera. `UiCameraBundle` has been removed. `Camera2dBundle` and `Camera3dBundle` now both default to rendering UI as part of their own render graphs. To disable UI rendering for a camera, disable it using the CameraUi component:
+
+```rust
+commands
+    .spawn_bundle(Camera3dBundle::default())
+    .insert(UiCameraConfig {
+        show_ui: false,
+        ..default()
+    })
+```
+
+```rust
+// 0.7
+camera.world_to_screen(transform, world_position);
+// 0.8
+camera.world_to_viewport(transform, world_position);
+```
+
+### [Visibilty Inheritance, universal ComputedVisibility and RenderLayers support](https://github.com/bevyengine/bevy/pull/5310)
+
+`Visibility` is now propagated into children in a similar way to `Transform`. Root elements of a hierarchy must now contain [`Visibility`] and [`ComputedVisiblity`] for visibility propagation to work.
+
+[`SpatialBundle`] and [`VisibilityBundle`] have been added for convenience.
+
+If you were previously reading `Visibility::is_visible` as the "actual visibility" for sprites or lights, use `ComputedVisibilty::is_visible()` instead:
```

Comment:
~~~suggestion
[`SpatialBundle`] and [`VisibilityBundle`] have been added for convenience.

```rust
// 0.7
commands
    .spawn_bundle(TransformBundle::default())
    .with_children(/* ... */)

// 0.8
commands
    .spawn_bundle(SpatialBundle::default())
    .with_children(/* ... */)
```

If you were previously reading `Visibility::is_visible` as the "actual visibility" for sprites or lights, use `ComputedVisibilty::is_visible()` instead:
~~~

I'm sort of on the fence about this, so feel free to disagree/veto. The current version is just missing the simple " `TransformBundle` users can just use `SpatialBundle` now " that some users might like to see. A sentence saying something similar would take up less space.

Reply:
Since it's already pretty long I'll go with the short sentence